src/frontend/screens/Settings/sections/GamesSettings/index.tsx

```diff
@@ -47,18 +47,26 @@ type Props = {
 export default function GamesSettings({ useDetails = true }: Props) {
   const { t } = useTranslation()
   const { platform } = useContext(ContextProvider)
-  const { isDefault, gameInfo, isMacNative, isLinuxNative } =
-    useContext(SettingsContext)
+  const { isDefault, gameInfo, isLinuxNative } = useContext(SettingsContext)
   const [wineVersion] = useSetting('wineVersion', defaultWineVersion)
   const isLinux = platform === 'linux'
-  const isWin = platform === 'win32'
   const isCrossover = wineVersion?.type === 'crossover'
   const hasCloudSaves = gameInfo?.cloud_save_enabled
 
-  const nativeGame =
-    isWin ||
-    (isMacNative && gameInfo?.install.platform === 'Mac') ||
-    (isLinuxNative && gameInfo?.install.platform === 'linux')
+  const [nativeGame, setNativeGame] = useState(false)
+
+  useEffect(() => {
+    if (gameInfo) {
+      const getIsNative = async () => {
+        const isNative = await window.api.isNative({
+          appName: gameInfo?.app_name,
+          runner: gameInfo?.runner
+        })
+        setNativeGame(isNative)
+      }
+      getIsNative()
+    }
+  })
```

Comment:
One last thing, you need to add a empty array here otherwise it will run infinitely.
Everytime you add a useEffect always add an empty array or an array with the dependencies for it to check if they change then the useEffect will trigger again. In this case an empty array is fine:

```suggestion
  }, [])
```